{{cookiecutter.repo_name}}/{{cookiecutter.repo_name}}/templates/forms/password.jinja

```diff
@@ -1,9 +1,17 @@
-{% from 'components/util.jinja' import attributes %}
-{% macro password(name='password', value=None, event_name="", attrs={}) %}
-    <div x-data="password('update-{{ event_name | replace('_', '-') }}', '{{ value }}', 'password')" x-ref="input"
+{% from 'components/util.jinja' import attributes, get_event_name_from_id %}
+{% macro password_field(field, errors) %}
+    {{ password_widget(field.field.widget, value=field.value(), id=field.id_for_label, name=field.name)}}
+{% endmacro %}
+
+{% macro password_widget(widget, value=none, id="", name="") %}
+    {% set event_name = get_event_name_from_id(id) %}
+    <div
+        x-data="password('{{ event_name }}', '{{ value }}', 'password')"
+        x-ref="input"
         :class="active ? 'has-focus' : ''"
-         class="w-full z-10 flex items-center relative password">
-        <input  @focus="updateFlag" @blur="updateFlag" x-model.debounce.500ms="value" :type="type" name="{{ name }}" class="mr-8" {% if value != None %}value="{{ value }}"{% endif %} {{ attributes(attrs)}} />
+        class="w-full z-10 flex items-center relative password"
+    >
+        <input  @focus="updateFlag" @blur="updateFlag" x-model.debounce.500ms="value" :type="type" class="!pr-10" value="{{ value or ""}}" {{ attributes(widget.attrs)}} />
```

Comment:
Looks like this is missing `name="{{ name }}"`

Reply:
this should be included in attrs now